python/packages/autogen-ext/src/autogen_ext/models/openai/_openai_client.py

```diff
@@ -561,8 +554,7 @@ async def create(
             logprobs=logprobs,
         )
 
-        _add_usage(self._actual_usage, usage)
```

Comment:
I'm wondering what's the difference between these two...

Reply:
Traditionally actual vs total is a matter of if the tokens were cached or not. But since we don't have caching atm they should be identical

Reply:
Thanks @jackgerrits! I have updated the PR to use both `actual_usage` and `total_usage`.

Reply:
Thanks @jackgerrits! I have updated the PR to use both `actual_usage` and `total_usage`.